api/filtering/filtering.py

```diff
@@ -270,6 +272,20 @@ def _generic_filter_builder(builder_function, field_name, field_value, field_fil
         return _base_filter_builder(nullable_builder_function, field_name, field_value, field_filter, operation, spec)
 
 
+def _build_registered_with_per_reporter_filter(registered_with):
```

Comment:
Since this is also being called from other files, it should probably not start with an underscore:
```suggestion
def build_registered_with_per_reporter_filter(registered_with):
```